### DISCUSSION TITLE
Suggestions for managing Rare Species Reader user permissions

### DISCUSSION THREAD

**Discussion by ljwalker** [Symbiota/community-board#189] [Wish List]
From SERNEC user:

> For future development of rare species reading rights functionality, we wonder:
> 1. If the requests for rare species reading rights could be automated to include only institutions that host relevant material.
> 2.  Just like with traditional specimen loans, could rare species rights be time limited? We are growing a list of folks with such rights and are positive not all of them need continued access.  It would be considerably time-saving, if reading rights timed out at the end of a permitted period.
> 3. Just like with traditional specimen loans, could rare species reading rights be limited to select taxa, rather than the current blanket access to the entire collection?

**Comment by ljwalker** [comment]
A CCH2 user (and SEINet snapshot) would like improved notification system regarding user permissions, as well as the activity of those users:

> ... There are names listed as "Rare Plant Readers" whom I don't know. I also struggle to stay on top of the list of people who have access for Georeferencing

> ... Symbiota could automatically send us a list of user names to review every time we add someone to our set of [users]

**Reply by Alison-CenterforPlantDiversity** [reply]
Thank you, Lindsay! Given the increased pressure from malicious cyber attacks in general, I'm going to step up scrutiny of approved editors/readers of CCH2 data for my collection. Automated reminders to do so would be nice, but we can put this in our own calendar too.
